src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    public Rectangle(String color) {
+        super(color);
+    }
+
+    @Override
+    public double getArea() {
+        return width * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: triangle, area: " + getArea()
+                + " sq. units, firstLeg: " + width + " units, secondLeg: "
+                + height + " units, color: " + getColor());
```

Comment:
The message in the 'draw' method incorrectly states the figure as a triangle, while it should be a rectangle. Also, 'firstLeg' and 'secondLeg' should be replaced with 'width' and 'height' to accurately describe the properties of a rectangle.